### PR TITLE
adds some documentation

### PR DESCRIPTION
**Pull request by chroetz**
# Purpose of this PR

Add documentation about Units of certain variables, in particular related to DAC.

## Type of change

- [x] Refactoring

## Checklist:

- [X] My code follows the coding etiquette
- [X] I have performed a self-review of my own code
- [X] Changes are commented, particularly in hard-to-understand areas
- [X] I have updated the in-code documentation
- [X] I have adjusted reporting where it was needed
- [X] The model compiles and runs successfully (`Rscript start.R -q`)

### CODE REVIEW

**Comment by fschreyer** [comment]
Thanks!

I would replace ``for dac: $(2015)/tC(removed)`` by ``for dac: $(2015)/tC(captured)``. Removed could be interpreted as "stored".

**Comment by chroetz** [comment]
> I would replace for dac: $(2015)/tC(removed) by for dac: $(2015)/tC(captured). Removed could be interpreted as "stored".

True! I changed it

**Comment by chroetz** [comment]
I noticed that sometimes `yr` is used for the unit *year* and sometimes `a`...

File: core/input/generisdata_tech.prn [hunk 1]
```diff
@@ -182,7 +184,7 @@ inco0               0.01       21600
 mix0                   0           0
 eta                 1.00        1.00
 omf                 0.84       0.025
-omv                    0           0
+omv                    0
```
Comment (LaviniaBaumstark) [inline core/input/generisdata_tech.prn:187]: are you sure that here are all row correct? in .prn files we need to be careful with tabs

Comment (chroetz) [reply core/input/generisdata_tech.prn:187]: hm, pretty sure. `Rscript start.R -q` works. No value is treated as 0. We decided to remove the 0 as it is not used for dac.

**Review by LaviniaBaumstark — APPROVED** [review]

**Pull request merged**